src/changelog/2023/10/path-bug-fix.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Device Agent path bug fix
+date: 2023-10-26 90:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+Resolved a Device Agent bug that prevented the PATH environment variable from being passed to Node-RED instances, hindering the installation of nodes like node-red-serial-port when assigned to an application. For more details, see [GitHub Issue](https://github.com/FlowFuse/device-agent/issues/185).
```

Comment:
```suggestion
The Device Agent now passes the PATH environment variable to the Node-RED instances. Erroneously, the agent didn't capture it to expose to Node-RED again for usage. This prevented some nodes from being installed, like `node-red-serial-port` when assigned to an application. For more details, see [GitHub Issue](https://github.com/FlowFuse/device-agent/issues/185).
```

The first sentence isn't a sentence. Should start with "We" or something instead. Or what I suggested.